pessimistic-transaction.md

```diff
@@ -119,6 +119,22 @@ TiDB 在悲观事务模式下支持了 2 种隔离级别：
 
 2. 使用 [`SET TRANSACTION`](/sql-statements/sql-statement-set-transaction.md) 语句可将隔离级别设置为[读已提交隔离级别 (Read Committed)](/transaction-isolation-levels.md#读已提交隔离级别-read-committed)。
 
+## 悲观事务提交流程
+
+TiDB 悲观锁复用了乐观锁的两阶段提交逻辑，重点在 DML 执行时做了改造。
+
+![TiDB 悲观事务的提交流程](/media/troubleshooting-lock-pic-05.png)
```

Comment:
I mean renaming the image files.  The file names are `troubleshooting-lock-xxx.png`, but now they are not used in the lock troubleshooting document but used in another document.

Reply:
Done in https://github.com/pingcap/docs-cn/pull/8191/commits/e6e952321ea6e7e8ec97ae85f7093ec6eb9ed594. PTAL again